scripts/launcher/lib/utils/iso_dev.py

```diff
@@ -47,6 +47,26 @@ def mount_dir(self):
                 raise IsoDeviceException("To read mount dir you have to mount the ISO first.")
         return self._iso_mount.mount_dir
 
+    @property
+    def kernel(self):
```

Comment:
These two properties are not used, so you can remove them.

Reply:
I don't see a purpose of the first commit, is that a leftover from other changes?

Reply:
Turned out to be not needed in the end and while useful can still be added at any time, so we can definitely drop it.